hardware/pico_driver/src/pico_proxy.cc

```diff
@@ -0,0 +1,187 @@
+/* Copyright (c) 2017, United States Government, as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ *
+ * All rights reserved.
+ *
+ * The Astrobee platform is licensed under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Core ROS
+#include <ros/ros.h>
+#include <nodelet/nodelet.h>
+#include <pluginlib/class_list_macros.h>
+
+// Shared libraries
+#include <ff_util/ff_names.h>
+#include <ff_util/ff_nodelet.h>
+#include <config_reader/config_reader.h>
+
+#include <image_transport/image_transport.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
+// Messages
+#include <sensor_msgs/Image.h>
+#include <ff_msgs/PicoflexxIntermediateData.h>
+
+#include <string>
+#include <map>
+
+
+// Layer extraction
+std::vector <cv::Mat> layers(4);
```

Comment:
Is there a reason this isn't a private data member?

Reply:
It crashed if I made it private :/ I don't know why

Reply:
Weird, okay let's leave it alone then.